WcaOnRails/config/environments/development.rb

```diff
@@ -52,9 +52,9 @@
   config.assets.raise_runtime_errors = true
 
   # One can activate/test the fallback to English using :
-  # config.i18n.fallbacks = true
+  config.i18n.fallbacks = true
```

Comment:
Given the infrastructure we have to detect missing keys, I decided to turn on the fallback even in development mode: I don't know if it was a bug from rails before, but since the gems update it's a pain to develop in a foreign language if some keys are missing!